lib/sqlalchemy/sql/compiler.py

```diff
@@ -5210,8 +5210,9 @@ def visit_drop_constraint(self, drop, **kw):
                 "Can't emit DROP CONSTRAINT for constraint %r; "
                 "it has no name" % drop.element
             )
-        return "ALTER TABLE %s DROP CONSTRAINT %s%s" % (
+        return "ALTER TABLE %s DROP CONSTRAINT %s%s%s" % (
             self.preparer.format_table(drop.element.table),
+            drop.if_exists and "IF EXISTS " or "",
             formatted_name,
             drop.cascade and " CASCADE" or "",
```

Comment:
these could be replaced with inline if-else (these probably date back to a python version that did not yet support it )

Reply:
Thanks! I tried to remain consistent with the existing patterns rather than diverge for my changes.
Origin of this pattern https://github.com/sqlalchemy/sqlalchemy/commit/cfa1a42328b406d0f494672f0b268890471d6d2e
If it’s desirable to convert to newer patterns, happy to do so!

Reply:
not really required, but I think it's safe to convert to the new syntax since we are modifying this part anyway.

If you make the change, could you also add a changelog for this pr?

Reply:
Changes updated.
Changelog added as well - wasn't really certain if the location (2.0) and tags we correct - I was basing it off the opened ticket labels.

Reply:
look great, thanks!